source/Cosmos.System2/FileSystem/Disk.cs

```diff
@@ -265,6 +274,7 @@ public void MountPartition(int index)
             var part = Partitions[index];
             //Don't remount
             if (MountedPartitions[index] != null)
+            //if (part.MountedFS != null) // Thank to ME i saved your time
```

Comment:
If the old check is wrong please remove it

Reply:
I removed that